src/ghostfragment/capping/DCLC.cpp

```diff
@@ -0,0 +1,128 @@
+#include "capping.hpp"
+#include <ghostfragment/property_types/capped.hpp>
+#include <ghostfragment/property_types/connectivity_table.hpp>
+#include <src/ghostfragment/connectivity/connectivity.hpp>
+
+namespace ghostfragment::capping {
+
+using my_pt      = ghostfragment::pt::Capped;
+using connect_pt = ghostfragment::ConnectivityTable;
+using traits_t   = ghostfragment::pt::CappedTraits;
+using atom_type  = traits_t::result_type::value_type::atom_type;
+
+
+// Computes the average X-C bond length in the molecule, where X is in
+// the fragment and C is the cap. Takes in Nuclei m, and the atomic
+// numbers of X and C.
+double average_bond_length(chemist::Nuclei m,
+const chemist::topology::ConnectivityTable connections,
```

Comment:
```suggestion
const chemist::topology::ConnectivityTable& connections,
```
Without the `&` it'll copy the connections.